docs/snippets/license_preamble.txt

```diff
@@ -0,0 +1,20 @@
+This file is part of %package_name%.
+
+Developed for the LSST Data Management System.
+This product includes software developed by the LSST Project
+(http://www.lsst.org).
+See the COPYRIGHT file at the top-level directory of this distribution
```

Comment:
There does seem to be an option to add a line here along the lines of:
```
Copyright LSST Data Management Contributing Institutions.
See the COPYRIGHT...
```
It's not a legal statement but it does give the reader a hint of what will be found in the COPYRIGHT file.